### PR TITLE
Changes IO classes members to use project naming scheme.

### PR DESCRIPTION
**Pull request by tomsons26**

### CODE REVIEW

**Review by ghost — DISMISSED** [review]

File: src/game/io/pkpipe.h [hunk 1]
```diff
@@ -39,14 +39,14 @@ class PKPipe : public Pipe
...
-    int m_encryptedKeyLength;
-    int m_carryOver;
+    BOOL m_ChangeKey;
+    Straw &m_CryptRandom;
+    BlowPipe m_BlowFish;
+    PipeControl m_Mode;
+    PKey *m_RSAKey;
+    int m_CryptoBuffer[256];
+    int m_EncryptedKeyLength;
+    int m_CarryOver;
```
Comment (tomsons26) [inline src/game/io/pkpipe.h]: missed this

File: src/game/io/lcwstraw.h [hunk 2]
```diff
@@ -32,14 +32,14 @@ class LCWStraw : public Straw
...
-    StrawControl m_mode;
-    int m_carryOver;
-    uint8_t *m_inBuffer;
-    uint8_t *m_outBuffer;
-    int m_blockSize;
-    int m_compressedBlockSize;
-    int16_t m_compressedBytes;
-    int16_t m_uncompressedBytes;
+    StrawControl m_Mode;
+    int m_CarryOver;
```
Comment (tomsons26) [inline src/game/io/lcwstraw.h]: missed this

File: src/game/io/blowstraw.h [hunk 3]
```diff
@@ -24,18 +24,18 @@
...
     void Key(void *key, int key_size);
 
 private:
-    BlowfishEngine *m_blowFish;
-    char m_currentBlock[BF_BLOCKSIZE];
-    int m_carryOver;
-    StrawControl m_mode;
+    BlowfishEngine *m_BlowFish;
+    char m_CurrentBlock[BF_BLOCKSIZE];
+    int m_CarryOver;
```
Comment (tomsons26) [inline src/game/io/blowstraw.h]: missed this

File: src/game/io/blowpipe.h [hunk 4]
```diff
@@ -33,10 +33,10 @@ class BlowPipe : public Pipe
     void Key(void *key, int key_size);
 
 private:
-    BlowfishEngine *m_blowFish;
-    char m_currentBlock[BF_BLOCKSIZE];
-    int m_carryOver;
-    PipeControl m_mode;
+    BlowfishEngine *m_BlowFish;
+    char m_CurrentBlock[BF_BLOCKSIZE];
+    int m_CarryOver;
```
Comment (tomsons26) [inline src/game/io/blowpipe.h]: missed this

**Review by OmniBlade — COMMENTED** [review]
I'm generally against the changes to the files that are common with Thyme as it makes it harder to backport fixes between the two code bases and makes it harder to work out what has changed from a diff. The fixes to files not shared with it are fine though.

**Review by ghost — APPROVED** [review]

**Pull request merged**